pkg/datarepair/checker/checker.go

```diff
@@ -108,19 +108,21 @@ func (c *checker) IdentifyInjuredSegments(ctx context.Context) (err error) {
 	return err
 }
 
-// returns the indices of offline and online nodes
-func (c *checker) offlineNodes(ctx context.Context, nodeIDs []dht.NodeID) (offline []int32, err error) {
+// returns the indices of offline and online nodeIds
+func (c *checker) offlineOnlineNodes(ctx context.Context, nodeIDs []dht.NodeID) (offline []int32, online []string, err error) {
```

Comment:
Nit - `offline` here is a slice of indices and  `online` is a slice of node IDs, but the similarity of the names implies two subsets of the same data. Maybe this ambiguity could be solved by giving them names like `lostPieces`/`missingPieces` and `onlineNodes`

Reply:
Also, a separate note that doesn't really involve this PR, but this would be a good spot to update node uptime stats in statdb. That way we have additional uptime information even when nodes aren't being audited